delocate/libsana.py

```diff
@@ -243,6 +244,140 @@ def walk_directory(
                 yield library_path
 
 
+def _tree_libs_from_libraries(
+    libraries: Iterable[str],
+    *,
+    lib_filt_func: Callable[[str], bool],
+    copy_filt_func: Callable[[str], bool],
+    executable_path: Optional[str] = None,
+    ignore_missing: bool = False,
+) -> Dict[str, Dict[str, str]]:
+    """ Return an analysis of the dependencies of ``libraries``.
+
+    libraries : iterable of str
+        The paths to the libraries to find dependencies of.
```

Comment:
Some comment here that all subsequent arguments are keyword-only?

Reply:
See: https://github.com/numpy/numpydoc/issues/330

Reply:
I've added the `keyword-only` attribute to the relevant parameters.